test/dynamics_lorenz_test

```diff
@@ -0,0 +1,76 @@
+import pypose.module.dynamics as ppmd
```

Comment:
Perhaps we can remove this example.  In terms of nonlinear dynamics examples, cart-pole seems sufficient.

Reply:
I thought that the idea was to have an example that doesn't require learning, and an example that requires learning. If not I can delete the example.